webapp/sources/ldap-inventory/inventory-repository/src/main/scala/com/normation/inventory/ldap/core/ReadOnlySoftwareInventoryDAOImpl.scala

```diff
@@ -123,18 +123,24 @@ class ReadOnlySoftwareDAOImpl(
 
       // fetch all nodes
       nodes        = con.searchSub(nodeBaseSearch, IS(OC_NODE), A_NODE_UUID)
-      batchedNodes = nodes.grouped(50).toSeq
-
+      batchedNodes = nodes.grouped(25).toSeq
```

Comment:
can you put the `25` in a var and use that var in log? I'm almost (to be fair with me) that I would forget half of the change if I ever touch that